app/config/sculpin_site.yml

```diff
@@ -19,6 +19,9 @@ subspecies:
     confoo:
         common: "ConFoo"
         latin: "fooianum"
+    darkmira:
+        common: "Darkmira PHP Tour"
+        latin: "tenebrisvultus"
```

Comment:
This seems to translate as "dark face". Is that what you were going for? My Portugeus and my French are pretty much non-existant,  so I'm having a hard time understanding what "Darkmira" is supposed to mean. 

I also don't think this is the right declension, but we can work on that next.

Reply:
This was almost 3 months ago so I don't remember exaclty what I was thinking at the time.

Mira, from mirar, is both Portugese and Spanish for "look" so I went with the literal translation of Dark + Look which comes out to `Ecce tenebris`.  I don't remember how I got to `tenebrisvultus` from that.  So I could change it to `eccetenebris`

Reply:
Actually I just plugged it in again and now it's showing that :man-shrugging::skin-tone-2:

![Google Translate screenshot](https://user-images.githubusercontent.com/1888809/53852205-05f2cd80-3f76-11e9-8f23-7b2185532b65.png)

Reply:
DarkmiraTour is a Brazil PHP Conference. A event with 3 days with more than 30 talks where international and national speakers will share their knowledge in English and Portuguese.
The 2019 edition is very special, for the first time ever we will have a plush ElePHPant exclusive to the DarkmiraTour PHP to support our diversity and inclusion actions.

More info: http://php.darkmiratour.rocks or our twitter: http://twitter.com/darkmiratour

Reply:
Hi @cookieguru and @philipsharp, there's no translation for Darkmira, it's only the name of the event, like a trademark :-)

Reply:
@cgrandval Of course Darkmira has no Latin translation, but since we need a [species name](https://github.com/philipsharp/afieldguidetoelephpants/blob/master/CONTRIBUTING.md#elephants) I just took the literal translation of the words instead :)

But since `Unity` is the official name we will instead use `unitas` as the species name.  I have also updated the `Name` field to include the official name.